ballerina/modules/types.any/tests/any_msg_pb.bal

```diff
@@ -0,0 +1,290 @@
+
+type mapTypeForTest map<anydata>;
+
+type bytesTypeForTest byte[];
+
+@protobuf:Descriptor {
+    value: ANNOTATED_MSG_DESC
+}
+public type AnnotatedMessageWithOneof record {|
+    float doubleData?;
+    float floatData?;
+    int uInt32Data?;
+    int uInt64Data?;
+    int int32Data?;
+    int int64Data?;
+    int fixed32Data?;
+    int fixed64Data?;
+    boolean booleanData?;
+    string stringData?;
+    MessageData messageData?;
+|};
+
+isolated function isValidAnnotatedmessagewithoneof(AnnotatedMessageWithOneof r) returns boolean {
+    int dataCount = 0;
+    if !(r?.doubleData is ()) {
```

Comment:
```suggestion
    if r?.doubleData !is () {
```

Reply:
We have to change this from the CLI level. Added a separate task https://github.com/ballerina-platform/ballerina-standard-library/issues/2952